src/sentry/tasks/integrations/github/open_pr_comment.py

```diff
@@ -102,3 +109,49 @@ def get_projects_and_filenames_from_source_file(
                 pr_filename.replace(code_mapping.source_root, code_mapping.stack_root)
             )
     return project_list, sentry_filenames
+
+
+def get_top_5_issues_by_count_for_file(
+    projects: List[Project], sentry_filenames: List[str]
+) -> list[dict[str, Any]]:
+    """Given a list of issue group ids, return a sublist of the top 5 ordered by event count"""
+    group_ids = list(
+        Group.objects.filter(
+            last_seen__gte=datetime.now() - timedelta(days=14),
```

Comment:
We're only searching issues from issues in the last 14 days?

Reply:
issues last seen within the last 14 days